pysidt/sidt.py

```diff
@@ -855,7 +862,7 @@ def estimate_uncertainty(self):
 
         if A.shape[1] != 1:
             node_uncertainties = (
-                np.diag(np.linalg.pinv((A.T @ A).toarray()))
+                np.diag(np.linalg.pinv((A.T @ W @ A).toarray()))
```

Comment:
I think this W matrix needs normalized to avoid affecting the uncertainties magnitude.